### DISCUSSION TITLE
Sunbird APP and Diksha App can't co-exist on same device.

### DISCUSSION THREAD

**Discussion by baljeetsingh** [Sunbird-Ed/Community#369] [Bugs] [answered]
We are using sunbird app white labeled app for our use case.

What we find is sunbird customized app and Diksha app can't co-exist on the same mobile app.

We are installing both form the playstore 
https://play.google.com/store/apps/details?id=org.pradigione.app&pli=1
https://play.google.com/store/apps/details?id=in.gov.diksha.app

<img src="https://user-images.githubusercontent.com/344965/214545511-3aa41a2d-7e44-4466-b77f-e7ca523c7dde.jpg" alt="drawing" width="300"/>

<img src="https://user-images.githubusercontent.com/344965/214556934-b2703d59-eb69-4673-8603-145693c0eb17.jpg" alt="drawing" width="300"/>

<img src="https://user-images.githubusercontent.com/344965/214556941-6db8d021-c1d5-41a8-8947-d38b03480e40.jpg" alt="" width="300"/>

**Comment by Krishnaj20** [comment, marked as answer]
@swayangjit @vinukumar-vs - Pls check and respond.

**Comment by swayangjit** [comment]
@baljeetsingh Please refer to this discussion link https://github.com/Sunbird-Ed/Community/discussions/12

**Reply by baljeetsingh** [reply]
thank you @swayangjit 

Are you referring us to use jenkins ?
Can you share details on how / where it is ?

![image](https://user-images.githubusercontent.com/344965/215992353-c3af5fc4-7bba-4d1b-8c97-b54eadfb508b.png)

**Reply by swayangjit** [reply]
@baljeetsingh That is not required, you have to update the plugin that is provided in the comment.

**Reply by nidhikachhawa24** [reply]
I've followed the following steps and was able to resolve the issue:

1. Added git repo link in build_config file `cordova-plugin=https://github.com/swayangjit/cordova-plugin-document-viewer.git`
2. Removed the plugin `cordova plugin remove cordova-plugin-document-viewer`
3. Added the plugin `cordova plugin add https://github.com/swayangjit/cordova-plugin-document-viewer.git`
4. Run `npm run ionic-build`

With the above steps I was able to install 2 flavours of Sunbird Mobile App

cc: @baljeetsingh @swayangjit
